product-mini/platforms/nuttx/wamr.mk

```diff
@@ -62,6 +66,35 @@ else ifeq (${WAMR_BUILD_TARGET}, XTENSA)
   CFLAGS += -DBUILD_TARGET_XTENSA
   INVOKE_NATIVE := invokeNative_xtensa.s
   AOT_RELOC := aot_reloc_xtensa.c
+else ifeq (${WAMR_BUILD_TARGET}, RISCV64)
+
+ifeq (${CONFIG_ARCH_FPU},y)
+  CFLAGS += -DBUILD_TARGET_RISCV64_LP64F
+  INVOKE_NATIVE += invokeNative_riscv64_lp64d.s
```

Comment:
LP64F isn't supported yet, not sure whether using invokeNative_riscv64_lp64d.s has potential issues?

Reply:
Removed,